lib/config/MoveConfig.js

```diff
@@ -103,8 +103,8 @@ const configSchema = Joi.object().keys({
   }),
   host: Joi.string().hostname(),
   https: Joi.object().keys({
-    key: Joi.binary().required(),
-    cert: Joi.binary().required(),
+    key: Joi.any().required(),
```

Comment:
...because we're now using the browser build in both client and server, which doesn't support `.binary()` (because that depends on the node.js `Buffer` type, IIRC).

please document this here.